.github/workflows/coverage-tests.yml

```diff
@@ -11,15 +11,15 @@ jobs:
   test-coverage:
     runs-on: self-hosted
     container:
-      image: localhost:5000/flagscale:cuda-12.4.1-python-3.12-torch-2.5.1-commit-a803e11 .
+      image: localhost:5000/flagscale:cuda-12.4.1-python-3.12-torch-2.5.1-commit-a803e11
```

Comment:
Can you find a way to define this image only once, so that it can be used by others?